iOS_SDK/OneSignalSDK/UnitTests/UnitTests.m

```diff
@@ -162,11 +162,14 @@ - (void)testBasicInitTest {
     
     NSLog(@"CHECKING LAST HTTP REQUEST");
     
+    NSString *model = [[UIDevice currentDevice] model];
+    NSString *deviceModel = [@"Simulator " stringByAppendingString:model];
```

Comment:
Let's add a comment here of what the final value should be for the iOS simulator. It should be "Simulator iOS" correct?